### DISCUSSION TITLE
Wifi connection?

### DISCUSSION THREAD

**Discussion by mloco** [doudar/SmartSpin2k#485] [Q&A] [answered]
Hey, i started setting up my smartspin today and i saw the WAP show up on initial power up, but then i fat-fingered the password to my home wifi network (i know, i know). I haven't been able to get the WAP to show up again since then, the device powers, and then the LED starts blinking red, nothing shows up in wifi settings. Is there a way to perform a reset and start the process again? Thanks in advance.

**Comment by eMadman** [comment, marked as answer]
If you wait a few minutes, it will end up in hotspot mode automatically

You can also reset the firmware with these steps:

Disconnect the power
Press and hold both shifter buttons
Reconnect the power
Wait 3 seconds
Release the buttons.

This should reset the firmware

**Reply by mloco** [reply]
thank you! the reset worked for me
